pkgs/development/python-modules/aioasuswrt/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, asyncssh
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-asyncio
+, pytest-cov
+, pytest-mock
+, pytest-runner
+, pytestCheckHook
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "aioasuswrt";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "kennedyshead";
+    repo = pname;
+    rev = "V${version}";
+    sha256 = "1vqihb5fgphb03867b90aayfnnrssda7vs97f84q7s4cwgjxi9lg";
+  };
+
+  nativeBuildInputs = [ pytest-runner ];
+
+  propagatedBuildInputs = [ asyncssh ];
+
+  checkInputs = [
+    pytest-asyncio
+    pytest-cov
+    pytest-mock
+    pytestCheckHook
+  ];
+
+  # Support for later Python releases: https://github.com/kennedyshead/aioasuswrt/issues/49
+  doCheck = pythonAtLeast "3.9";
```

Comment:
```suggestion
  disabled = pythonAtLeast "3.8";
```

Reply:
Note that this will make make `parse-requirements.py` not pick it up.

Reply:
Will not longer be needed with 13.2